source/docs/articles/wordpress/site-networks/getting-started.md

```diff
@@ -0,0 +1,90 @@
+---
+title: Up and Running with WordPress Site Networks
+description: Get started with WordPress Site Networks. Learn about the Pantheon WordPress Site Network upstream product, start developing, or import existing networks.
+---
+
+Once a Pantheon employee has given you access to your WordPress Site Network site, you can import an existing site network or start from scratch and begin developing your network on the platform.
+
+## Migrating Site Networks to Pantheon
+
+If your network already exists, follow the instructions in the [Migrate to Pantheon: WordPress Site Networks](/docs/articles/sites/migrate/wordpress-site-networks/) doc. 
+
+## Create a Network
+Before you begin, switch the Development environment to SFTP mode. Creating the network requires making modifications to the `wp-config.php` file, which is not editable in Git mode.
+
+### Create a Network with WP-CLI (Recommended)
+
+If you’re comfortable with the command line, Terminus and WP-CLI let you convert an existing WordPress environment to Multisite in one command: `wp core multisite-convert`
+
+Before you begin, make sure that:
+
+- Your WordPress environment is functioning as expected.
+- Terminus is configured locally, and you’re authenticated with Pantheon.
+
+Once you’ve confirmed these prerequisites, you can enable WordPress Multisite with one command.
+
+Subdirectory style:
+```bash
+terminus wp core multisite-convert --site=<pantheon-site> --env=dev
+```
+
+Subdomains style:
+```bash
+terminus wp core multisite-convert --site=<pantheon-site> --env=dev --subdomains
+```
+
+Optionally specify --title to use a custom title for your Multisite Network:
+
+```bash
+terminus wp core multisite-convert --site=<pantheon-site> --env=dev --title=”My Awesome Multisite Network”
+```
+
+For full compatibility with Pantheon, you’ll need to update DOMAIN_CURRENT_SITE to be set conditionally based on environment. Here is an example:
+
+https://gist.github.com/danielbachhuber/69c44664d4d63a6e19db
```

Comment:
Suggest including the code snippet within the doc contents, instead of linking to gist. Or embed the gist